src/sigstore.test.ts

```diff
@@ -65,8 +86,7 @@ describe('sign', () => {
 
   it('returns the correct envelope', async () => {
     const sig = await sign(payload);
-
-    expect(sig).toEqual(signedPayload);
+    expect(sig).toMatchSnapshot();
```

Comment:
Don't love these kinds of tests as I think it's easy for breaking changes to sneak past undetected but also painful to write out the entire json string here.